nuget/nunitlite/nunitlite.nuspec

```diff
@@ -46,6 +46,9 @@ How to use this package:
       <group targetFramework="net6.0">
         <dependency id="NUnit" version="[$version$]" />
       </group>
+      <group targetFramework="net7.0">
```

Comment:
@manfred-brands Wondering if I can ask your confirmation of my understanding here since you did the previous upgrades in this area. This would apply to the framework of the _consuming_ app I think, right?

Reply:
@stevenaw If the app consuming nunitlite targets any of the listed framework, it will automatically add a transient dependency on the packages listed here.